<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_Sign_page</class>
 <widget class="QMainWindow" name="Login_Sign_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget
{
	border-image: url(:/new/prefix1/Pics/login page bg 2.jpg);
}</string>
   </property>
   <widget class="QWidget" name="Login_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>230</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(62, 219, 240,0.9);

color: rgb(0, 0, 0);</string>
    </property>
    <layout class="QGridLayout" name="loginPage">
     <item row="1" column="0">
      <widget class="QLabel" name="PassWord_lable_2">
       <property name="text">
        <string> PassWord</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="Login_To_Sign_button">
       <property name="styleSheet">
        <string notr="true">background-color:rgba(26, 19, 212)</string>
       </property>
       <property name="text">
        <string>Dont have an account ? Sign Up</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Login_PassWord_input">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(119, 172, 241)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Login_button">
       <property name="styleSheet">
        <string notr="true">background-color:rgba(26, 19, 212)</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Login_UserName_input">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(119, 172, 241)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="UserName_lable_2">
       <property name="text">
        <string> UserName :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="SignUp_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>230</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,.7) ;

color: rgb(0, 0, 0);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="PassWord_lable">
       <property name="text">
        <string>PassWord</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Sign_PassWord_input2">
       <property name="styleSheet">
        <string notr="true">background-color:: rgba(162, 219, 250);

</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Sign_PassWord_input1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Sign_Email_input">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Email_lable">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="UserName_lable">
       <property name="text">
        <string>UserName</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="SignUp_Bottom">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 85, 127);</string>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QRadioButton" name="term_service_ratio">
       <property name="text">
        <string>Agree to our term and service</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="PassWord_lable_3">
       <property name="text">
        <string>PassWord</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Sign_UserName_input">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="sign_to_login_button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 85, 127);</string>
       </property>
       <property name="text">
        <string>Already Have an Account ? Login </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="Output">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(85, 85, 127, 0.8)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
